web/src/screens/settings/Releases.tsx

```diff
@@ -33,6 +33,45 @@ function ReleaseSettings() {
 
   const cancelModalButtonRef = useRef(null);
 
+  const getDurationLabel = (durationValue: number): string => {
+    const durationOptions: Record<number, string> = {
```

Comment:
Should add a "DELETE EVERYTHING EVER" option (in caps) and get rid of the old "Delete all releases" button, as someone will surely get confused and fuck up. It could occupy the value 0, for example.